template.js

```diff
@@ -336,11 +336,13 @@ function trackCommonData(postBody) {
         if (browserVersion) postBody.properties['$browser_version'] = browserVersion;
     }
 
-    let initialReferrer = getCookieValues('stape_mixpanel_initial_referrer')[0];
-    if (initialReferrer) postBody.properties['$initial_referrer'] = initialReferrer;
-    if (!initialReferrer) {
-        postBody.properties['$initial_referrer'] = postBody.properties['$referrer'] ? postBody.properties['$referrer'] : 'direct';
-        setCookie('stape_mixpanel_initial_referrer', makeString(postBody.properties['$initial_referrer']), cookieOptions);
+    if (!data.trackParametersRemove || !data.trackParametersRemove.contains('$initial_referrer')) {
```

Comment:
I am getting an error message: ```Object has no 'contains' property.```
Also ```data.trackParametersRemove``` structure will look like this ```[{"name":"$initial_referrer"}]```

Reply:
You're right, thanks. I've changed the statement